prowler/providers/kubernetes/kubernetes_provider_new.py

```diff
@@ -0,0 +1,94 @@
+import os
+import sys
+from typing import Any, Optional
+
+from colorama import Fore, Style
+from kubernetes import client, config
+
+from prowler.lib.logger import logger
+from prowler.providers.common.provider import CloudProvider
+
+
+class KubernetesProvider(CloudProvider):
```

Comment:
Make a `TODO` here to change `CloudProvider` to `Provider`, CC: @n4ch04

Reply:
Done